tests/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs

```diff
@@ -79,9 +79,8 @@ public async Task Post_WithManyOwnedEntity_ReturnsDatabaseNumberId(ApiFixture ap
                 .Which.Entity.keyId.Should().BeGreaterThan(0);
 
             queryResult.Should().NotBeNull();
-
-            // TODO: add odata controller to test include properly
-            //queryResult!.ToDto().OwnedEntities!.Single().Name.Should().Be(expectedOwnedName);
+            queryResult.ExtractResult().CountryLocalNames.Should().NotBeNull();
```

Comment:
I'm curious, not sure how you were able to retrieve/load the owned entity not having Odata working and set AutoExpand in this test project?

Reply:
I don't know the reason why it didn't work before, but now it works at this test runs successfully